src/Template/Element/sentences/sentence_and_translations.ctp

```diff
@@ -83,7 +83,7 @@ $sentenceUrl = $this->Url->build([
     </div>
     <div layout="column">
         <div layout="row" class="header">
-            <md-subheader flex class="ellipsis">
+            <md-subheader flex>
```

Comment:
Removing the `ellipsis` class causes the sentence number and owner written out vertically on the left border of the sentence box when the sentence menu is expanded. You can see this on the dev site.

I guess you need to replace the `ellipsis` class with `ng-if="!vm.isMenuExpanded"`.